debug/src/index.js

```diff
@@ -4,3 +4,9 @@ import 'preact/devtools';
 initDebug();
 
 export { resetPropWarnings } from './check-props';
+
+export {
+	getCurrentVNode,
+	getDisplayName,
+	getOwnerStack
+} from './component-stack';
```

Comment:
Mind updating the [types](https://github.com/preactjs/preact/blob/main/debug/src/index.d.ts) to match?